scripts/run-tap.js

```diff
@@ -11,11 +11,16 @@ if (process.env.CI) {
 }
 
 // Enable the `--experimental-sqlite` flag for Node.js ^22.5.0
-if (major === 22 && minor >= 5) {
+if ((major === 22 && minor >= 5) || major === 23) {
```

Comment:
`isVersionGreaterOrEqual("22.5.0", getSemverNodeVersion())`?

Reply:
Can not import TS function from JS